synapse/lib/stormtypes.py

```diff
@@ -6193,6 +6193,58 @@ class Layer(Prim):
                   ),
                   'returns': {'name': 'Yields', 'type': 'storm:node',
                               'desc': 'Yields nodes.', }}},
+
+        {'name': 'getEdges', 'desc': '''
+            Yield (n1iden, verb, n2iden) tuples for any light edges in the layer.
+
+            Example:
+                Iterate the light edges in ``$layer``::
+
+
+                for ($n1iden, $verb, $n2iden) in $layer.getEdges() {
```

Comment:
needs to be indented one more level, also there is an extra blank line above this